cmd_support_version.md

```diff
@@ -0,0 +1,252 @@
+| 198    | GEORADIUSBYMEMBER_RO | v1.1.12           |                                                              |
+| 199    | GEOSEARCH            | v2.6.0            |                                                              |
+| 200    | GEOSEARCHSTORE       | v2.6.0            | **Doc mistake, the err command: getsearchstore**(manual check) |
+| 201    | XADD                 | v2.2.0            |                                                              |
+| 202    | XDEL                 | v2.2.0            |                                                              |
+| 203    | XINFO                | v2.2.0            |                                                              |
+| 204    | XLEN                 | v2.2.0            |                                                              |
+| 205    | XRANGE               | v2.2.0            |                                                              |
+| 206    | XREAD                | v2.2.0            |                                                              |
+| 207    | XREVRANGE            | v2.2.0            |                                                              |
+| 208    | XTRIM                | v2.2.0            |                                                              |
+| 209    | XCLAIM               | -                 |                                                              |
+| 210    | XAUTOCLAIM           | -                 |                                                              |
+| 211    | XGROUP               | -                 | Not Support Since v2.7.0(manual check)                       |
+| 212    | XPENDING             | -                 |                                                              |
+| 213    | XREADGROUP           | -                 |                                                              |
+| 214    | XSETID               | v2.3.0            |                                                              |
+| 215    | BF.RESERVE           | v2.6.0            |                                                              |
+| 216    | BF.ADD               | v2.6.0            |                                                              |
+| 217    | BF.EXISTS            | v2.6.0            |                                                              |
+| 218    | BF.CARD              | v2.6.0            |                                                              |
+| 219    | BF.INFO              | v2.6.0            |                                                              |
+| 220    | BF.MADD              | v2.6.0            |                                                              |
+| 221    | BF.INSERT            | v2.7.0            |                                                              |
+| 222    | BF.MEXISTS           | v2.6.0            | **Doc mistake, the err command: bf.mexist**(manual check)    |
```

Comment:
Would you mind also fixing the doc?

Reply:
@mapleFU yeah, I will fix the doc in my pr!